VK_App/VK_App/Model/UserPhoto.swift

```diff
@@ -5,6 +5,12 @@ import RealmSwift
 
 /// Фотографии пользователя
 final class UserPhoto: Object, Decodable {
+    /// Высота
+    let height: Int
+    /// Ширина
+    let width: Int
     /// адрес на фото друга
     @Persisted var url: String
+    /// Cоотношение сторон
+    var aspectRatio: CGFloat { CGFloat(height) / CGFloat(width) }
```

Comment:
```suggestion
    var aspectRatio: CGFloat { CGFloat(height) / CGFloat(width) }
```
нет защиты от деления на 0